src/plugins/foxPage/hooks/getGovernanceData.ts

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios'
+import { useEffect, useState } from 'react'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+
+type BoardroomGovernanceData = Array<{
+  currentState: string
+  title: string
+  choices: Array<string>
+  results: Array<{ total: number; choice: number }>
+  refId: string
+}>
+
+const BOARDROOM_API_BASE_URL = 'https://api.boardroom.info/v1/protocols/shapeshift'
+export const BOARDROOM_APP_BASE_URL = 'https://boardroom.io/shapeshift'
+
+const parseGovernanceData = (governanceData: BoardroomGovernanceData) => {
+  const activeProposals = governanceData.filter(data => data.currentState === 'active')
+  const proposals = activeProposals.length ? activeProposals : [governanceData[0]]
+
+  return proposals.map(({ title, choices, results, refId }) => {
+    const totalResults = results.reduce((acc, currentResult) => {
+      acc = acc.plus(currentResult.total)
+      return acc
+    }, bnOrZero('0'))
+
+    return {
+      refId,
+      title,
+      choices,
+      results: results.map(result => ({
+        absolute: result.total,
+        percent: bnOrZero(result.total).div(totalResults).toString(),
+      })),
+    }
+  })
+}
+
+export const useGetGovernanceData = () => {
```

Comment:
non blocking comment - please be aware hooks are *not* singletons, and if you're calling this in multiple places we'll be banging on this api a lot.